keyboards/handwired/amigopunk/readme.md

```diff
@@ -0,0 +1,7 @@
+# amigopunk
```

Comment:
This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).

Reply:
Done, leme know if I should squash everything.